samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <AppRoute />
-  </React.StrictMode>
```

Comment:
Why did we remove these code ?

Reply:
Restored as per React 18.2.0 recommendation.